src/styles/main.scss

```diff
@@ -1,3 +1,45 @@
+$block-size: 300px;
+
+$colors:
+  "1" rgb(100%, 0, 0),
+  "2" rgb(80%, 0, 0),
+  "3" rgb(60%, 0, 0),
+  "4" rgb(40%, 0, 0),
+  "5" rgb(20%, 0, 0),
+  "6" rgb(0, 0, 0);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
 body {
   margin: 0;
 }
+
+.snake {
+  display: grid;
+
+  &__block {
+    display: grid;
+    place-items: center;
+    height: $block-size;
+    min-width: $block-size;
+
+    @each $num, $color in $colors {
+      &--#{$num} {
+        background-color: $color;
+      }
+    }
+  }
+
+  @media (min-width: 600px) {
```

Comment:
If you want, you could calculate min-width for @media, you already have $block-size.